packages/cart/src/models/line-item.ts

```diff
@@ -47,7 +50,7 @@ export default class LineItem {
   @Property({ columnType: "text", nullable: true })
   thumbnail?: string | null
 
-  @Property({ columnType: "text" })
+  @Property({ columnType: "integer" })
```

Comment:
question: do we need the serializer here? should we use numeric?

Reply:
Why do you find the numeric type needed here?

Reply:
would make sense to allow quantity to not be just integers? The alternative would be more complex and associate it with a unit of measure or any other "scale" to be generic.

Reply:
my bad, integer should be good here

Reply:
I like this idea @carlos-r-l-rodrigues, but for the sake of keeping it simple initially, I would like to punt this to a future itereation.